rest-api/config/options/all.xml

```diff
@@ -130,6 +130,106 @@
       <facet-option>frequency-order</facet-option>
     </range>
   </constraint>
+  <constraint name="attachment">
+    <custom>
+      <parse apply="parse-structured" ns="http://marklogic.com/grouping-constraint" at="/ext/mlpm_modules/ml-constraints/grouping-constraint.xqy"/>
+      <start-facet apply="start" ns="http://marklogic.com/grouping-constraint" at="/ext/mlpm_modules/ml-constraints/grouping-constraint.xqy"/>
+      <finish-facet apply="finish" ns="http://marklogic.com/grouping-constraint" at="/ext/mlpm_modules/ml-constraints/grouping-constraint.xqy"/>
+      <facet-option>limit=25</facet-option>
+      <facet-option>frequency-order</facet-option>
+      <facet-option>descending</facet-option>
+      <facet-option>any</facet-option>
```

Comment:
very very very important, if you miss adding this element... ml-constraints will always return an empty cts:or-query.